Loop/View Models/SettingsViewModel.swift

```diff
@@ -99,6 +99,10 @@ public class SettingsViewModel: ObservableObject {
        }
     }
 
+    var showDeleteTestData: Bool {
```

Comment:
Thinking it might make sense to rename this to `hideDeleteTestData`. Any opinions?